linkcheck/configuration/__init__.py

```diff
@@ -428,43 +428,45 @@ def get_user_config():
     return userconf
 
 
-def get_gconf_http_proxy():
-    """Return host:port for GConf HTTP proxy if found, else None."""
+def get_dconf_http_proxy():
+    """Return host:port for dconf HTTP proxy if found, else None."""
     try:
-        import gconf
+        import gi
+        from gi.repository import Gio
     except ImportError:
         return None
     try:
-        client = gconf.client_get_default()
-        if client.get_bool("/system/http_proxy/use_http_proxy"):
-            host = client.get_string("/system/http_proxy/host")
-            port = client.get_int("/system/http_proxy/port")
+        settings = Gio.Settings.new("org.gnome.system.proxy.http")
```

Comment:
Ok, so this uses gsettings, not dconf.  There's a subtle difference.

gsettings can use different backends, and dconf is only one of them (which happens to be the default backend, unless you're running in a flatpak app container).

If we use gsettings, we'd better refer to it (rather than dconf) in documentation and function names.  (Although maybe a more user-friendly way would be to say we support GNOME's proxy configuration without going into details?)

Reply:
> If we use gsettings, we'd better refer to it (rather than dconf) in documentation and function names

and error messages!  I forgot to mention error messages.

Reply:
GNOME is better here and matches the use of KDE.